plugins/blocks/metadataExport/block.tpl

```diff
@@ -0,0 +1,49 @@
+{**
+ * plugins/blocks/metadataExport/block.tpl
+ *
+ * Copyright (c) 2013-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Common site sidebar menu -- metadata export.
+ *
+ *}
+
+{if $articleId}
+	{assign var="elementId" value=$articleId}
+	{assign var="op" value="exportArticle"}
+{elseif $issueId}
+	{assign var="elementId" value=$issueId}
+	{assign var="op" value="exportIssue"}
+{elseif $journalId}
+	{assign var="elementId" value=$journalId}
+	{assign var="op" value="exportJournal"}
+{/if}
+
+<div class="block">
+	<br />
+	<form action="{url page="metadataExport"}">
+
+	{if $journalId}
+		<span class="blockTitle">{translate key="plugins.block.metadataExport.title"}</span>
+		<input type='radio' name='exportScope' value='exportJournal' checked />{translate key="plugins.block.metadataExport.exportJournal"}<br />
+	{/if}
+
+	{if $articleId || $issueId}
+		<input type='radio' name='exportScope' value={$op} />{translate key="plugins.block.metadataExport.$op"}<br />
```

Comment:
Be careful with unquoted / unescaped template variables -- should be `value="{$op|escape}"` for safety/consistency.